mitosheet/src/components/taskpanes/ControlPanel/SummaryStatsTab/ColumnSummaryGraph.tsx

```diff
@@ -21,6 +21,7 @@ function ColumnSummaryGraph(props: ColumnSummaryGraphProps): JSX.Element {
         const _graphHTMLAndScript = await props.mitoAPI.getGraph(
             GraphType.SUMMARY_STAT,
             props.selectedSheetIndex,
+            false, // We use a different filtering mechanism for column summary graphs, so we just default to false here
```

Comment:
I don't think we need to refactor it now, but note that we really kinda have two different APIs here - and merging them was not it in some ways...